src/Microsoft.DotNet.Build.Tasks/Targets/publishtest.targets

```diff
@@ -36,8 +31,6 @@
   </ItemGroup>
 
   <Target Name="CopyTestToTestDirectory"
-    Inputs="@(PackagesConfig);$(TestRuntimePackageConfig);@(ReferenceCopyLocalPaths);@(Content);@(IntermediateAssembly);@(_DebugSymbolsIntermediatePath)"
```

Comment:
I think it is probably worth actually using he same Inputs/Outputs the copy as you are doing for the Run.